phpBB/phpbb/event/dispatcher.php

```diff
@@ -13,11 +13,11 @@
 
 namespace phpbb\event;
 
-use Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher;
+use Symfony\Component\EventDispatcher\EventDispatcher;
 use Symfony\Component\EventDispatcher\Event;
 
 /**
-* Extension of the Symfony2 EventDispatcher
+* Extension of the Symfony3.3 EventDispatcher
```

Comment:
Why 3.3? Pretty sure we're using 3.4.

Reply:
Somehow I thought it was added in 3.3, but the `ContainerAwareEventDispatcher` we were using is marked deprecated since 3.3
So I thought I would update that, but now it seems more logical to remove the version all together..